hub/auto/unstructured/image_classification.py

```diff
@@ -46,7 +68,7 @@ def _set_name_from_path(path: Path) -> str:
 
 
 class ImageClassification(UnstructuredDataset):
```

Comment:
Audio, video support should not be under ImageClassification. One thing you can try is renaming this class to something else and then subclass it for audio and video.